Simple-Login-System-Exercise/index.php

```diff
@@ -0,0 +1,12 @@
+<?php 
+
+spl_autoload_register(function($class)
+{
+    $path = __DIR__ . '/' . str_replace('\\', '/', $class) . '.php';
+    require_once $path;
+});
+
+use src\Views\LoginView;
```

Comment:
[Please refer to this link.](https://www.php-fig.org/psr/psr-12/#3-declare-statements-namespace-and-import-statements)